tests/bluetooth/tester/src/btp_mcp.c

```diff
@@ -1317,3 +1320,224 @@ uint8_t tester_unregister_mcp(void)
 {
 	return BTP_STATUS_SUCCESS;
 }
+
+/* Media Control Service */
+static uint8_t mcs_supported_commands(const void *cmd, uint16_t cmd_len, void *rsp,
+				      uint16_t *rsp_len)
+{
+	struct btp_mcs_read_supported_commands_rp *rp = rsp;
+
+	/* octet 0 */
+	tester_set_bit(rp->data, BTP_MCS_READ_SUPPORTED_COMMANDS);
+	tester_set_bit(rp->data, BTP_MCS_CMD_SEND);
+	tester_set_bit(rp->data, BTP_MCS_MEDIA_STATE_GET);
+	tester_set_bit(rp->data, BTP_MCS_CURRENT_TRACK_OBJ_ID_GET);
+	tester_set_bit(rp->data, BTP_MCS_NEXT_TRACK_OBJ_ID_GET);
+
+	*rsp_len = sizeof(*rp) + 1;
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t mcs_cmd_send(const void *cmd, uint16_t cmd_len, void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_mcs_send_cmd *cp = cmd;
+	struct mpl_cmd mcp_cmd;
+	int err;
+
+	LOG_DBG("MCS Send Command");
+
+	mcp_cmd.opcode = cp->opcode;
+	mcp_cmd.use_param = cp->use_param;
+	mcp_cmd.param = sys_le32_to_cpu(cp->param);
+
+	err = media_proxy_ctrl_send_command(mcs_media_player, &mcp_cmd);
+	if (err) {
+		return BTP_STATUS_FAILED;
+	}
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static void btp_send_mcs_media_state_ev(uint8_t state)
+{
+	struct btp_mcs_media_state_ev ev;
+
+	ev.state = state;
+
+	tester_event(BTP_SERVICE_ID_GMCS, BTP_MCS_MEDIA_STATE_EV, &ev, sizeof(ev));
+}
+
+static void btp_send_mcs_current_track_obj_id_ev(uint8_t status, uint64_t id)
+{
+	struct btp_mcs_current_track_obj_id_ev ev;
+
+	ev.status = status;
+	sys_put_le48(id, ev.id);
+
+	tester_event(BTP_SERVICE_ID_GMCS, BTP_MCS_CURRENT_TRACK_OBJ_ID_EV, &ev, sizeof(ev));
+}
+
+static void btp_send_mcs_next_track_obj_id_ev(uint8_t status, uint64_t id)
+{
+	struct btp_mcs_next_track_obj_id_ev ev;
+
+	ev.status = status;
+	sys_put_le48(id, ev.id);
+
+	tester_event(BTP_SERVICE_ID_GMCS, BTP_MCS_NEXT_TRACK_OBJ_ID_EV, &ev, sizeof(ev));
+}
+
+static void btp_send_media_player_command_ev(uint8_t status, const struct mpl_cmd *cmd)
+{
+	struct btp_mcs_command_ev ev;
+
+	ev.status = status;
+	ev.opcode = cmd->opcode;
+	ev.use_param = cmd->use_param;
+	ev.param = sys_cpu_to_le32(cmd->param);
```

Comment:
In the case that `use_param` is 0x00 (`false`), is `cmd->param` always initialized?

Reply:
Nah, just wanted to ensure we weren't calling `sys_cpu_to_le32` with uninitialized values :)